### ISSUE TITLE
scan-build fails on Python 3 with decoding error

### ISSUE THREAD

**Issue by apollo13** [rizsotto/scan-build#95]
Running scan-build against a build where the source files are iso-8859-15 results in the following error:
```
scan-build: Run 'scan-view /home/gitlab-runner/builds/1ce604f5/0/BAP/bap_server/tmp/scan-build-2017-12-27-11-39-48-717658-pd2twp3b' to examine bug reports.
Traceback (most recent call last):
  File "/usr/local/bin/scan-build", line 11, in <module>
    sys.exit(scan_build())
  File "/usr/local/lib/python3.5/dist-packages/libscanbuild/__init__.py", line 126, in wrapper
    return function()
  File "/usr/local/lib/python3.5/dist-packages/libscanbuild/analyze.py", line 71, in scan_build
    number_of_bugs = document(args)
  File "/usr/local/lib/python3.5/dist-packages/libscanbuild/report.py", line 41, in document
    for bug in read_bugs(args.output, html_reports_available):
  File "/usr/local/lib/python3.5/dist-packages/libscanbuild/report.py", line 283, in read_bugs
    for bug in parser(bug_file):
  File "/usr/local/lib/python3.5/dist-packages/libscanbuild/report.py", line 331, in parse_bug_html
    for line in handler.readlines():
  File "/usr/lib/python3.5/codecs.py", line 321, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xfc in position 6777: invalid start byte
```

**Comment by rizsotto** [comment]
Thanks for the report, will look at it next year.

**Comment by rizsotto** [comment]
Fixed in 2.0.11. Please reopen if it's still failing for you.
